examples/01_plotting/plot_3d_map_to_surface_projection.py

```diff
@@ -0,0 +1,55 @@
+"""
+Making a surface plot of a 3D statistical map
+=============================================
+
+project a 3D statistical map onto a cortical mesh using
+:func:`nilearn.surface.vol_to_surf`. Display a surface plot of the projected
+map using :func:`nilearn.plotting.plot_surf_stat_map`.
+
+"""
+
+##############################################################################
+# Get a statistical map
+# ---------------------
+
+from nilearn import datasets
+
+localizer_dataset = datasets.fetch_localizer_button_task()
+localizer_tmap = localizer_dataset.tmaps[0]
+
+##############################################################################
+# Get a cortical mesh
+# -------------------
+
+fsaverage = datasets.fetch_surf_fsaverage5()
+
+##############################################################################
+# Sample the 3D data around each node of the mesh
+# -----------------------------------------------
+
+from nilearn import surface
+
+texture = surface.vol_to_surf(localizer_tmap, fsaverage.pial_right)
+
+##############################################################################
+# Plot the result
+# ---------------
+
```

Comment:
Just to be aware that could you please comment saying that plot_surf_stat_map works  ?

NOTE: Example needs matplotlib version higher than 1.3.1.

Reply:
I'm sorry what should the comment say?

Reply:
I added your NOTE to the docstring